themes/default/content/blog/pulumi-release-notes-0.57/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Pulumi release notes: Milestone 58"
```

Comment:
If we are going to frame this is being about a "milestone", we likely should talk about what milestones are - and point at where to understand more about them - possibly by pointing to https://github.com/pulumi/pulumi/wiki/Roadmap which talks about milestones in the intro, and updating the recent milestones to match what is in these blog posts?

That said - I'm not sure we *want* to focus on the inside baseball of milestones.  Perhaps just "What's New in Pulumi: June 9th, 2021"? or something along those lines?  That gives us flexibility to *not* tie this too strictly to milestones going forward if they aren't the right cadence.

Reply:
YEah I 100% agree! We don't really use milestones outside of Pulumi internal tracking - we regularly make releases outside of them